cmake/Platform/Windows/Packaging/PostInstallSetup.wxs

```diff
@@ -61,5 +62,11 @@
             Execute="deferred"
             Impersonate="no"/>
 
+        <CustomAction Id="UnRegisterEngine"
+            ExeCommand="&quot;[SystemFolder]cmd.exe&quot; /C @&quot;[root.scripts]o3de.bat&quot; register --remove --this-engine"
```

Comment:
Just want to confirm this won't return non-zero if the engine isn't registered, right?

Reply:
I'll go ahead and ignore the return, that'd probably be best so it doesn't fail the uninstall process.

Reply:
Cool, thanks for confirming. Agreed that adding the ignore would be the best safeguard against unrelated command failures.